packages/meditrak-app/app/assessment/Question.js

```diff
@@ -39,6 +40,7 @@ const QUESTION_TYPES = {
   Number: NumberQuestion,
   Photo: PhotoQuestion,
   Radio: RadioQuestion,
+  Selection: SelectionQuestion,
```

Comment:
I don't think there we support any question type like the mock up (https://app.zenhub.com/files/232001970/fcaeb48f-54bc-4a24-ac37-2089a0829ae9/download). The closest we have is Radio question. So I created a new type of question above

Reply:
As discussed with Edwin, we will use the existing Radio question.